syntax = "proto3";

package idvalid.tenant;

message Tenant {
  uint64 id = 1;
  string subid = 2;
  string name = 3;
  bool is_active = 4;
}

message TenantUser {
  uint64 tenant_id = 1;
  uint64 user_id = 2;
  bool is_owner = 3;
  bool is_registered = 4;
  bool is_active = 5;
}
